tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -134,59 +134,79 @@ public function testFixProvider()
                 '<?php function xyz(
                     $a=10,      //comment1
                     $b=20,      //comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'must keep align comments (2)' => [
                 '<?php function xyz(
                     $a=10,  //comment1
                     $b=2000,//comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'multiline comments also must be ignored (I)' => [
-                '<?php function xyz(
+                <<<'EXPECTED'
+<?php function xyz(
```

Comment:
Changing the indent of a nowdoc changes the value of the content. In this case adding white space before the open tag effectivly changing the test case. I'm not sure why this string needs changing to nowdoc here?